app/models/activity.rb

```diff
@@ -139,4 +139,8 @@ def iati_identifier
       parent_identifiers << parent.identifier
     }.push(identifier).join("-")
   end
+
+  def transaction_total
```

Comment:
How about `transactions_total` to match the association name `transactions` ?

Reply:
Good call 👍